packages/storybook-html/src/components/tabs/tabs.stories.js

```diff
@@ -1,19 +1,212 @@
-import tabsHTML from './tabs.html';
+// import scripts
+import { Tabs } from '@uqds/tabs/src/js/tabs';
+
+// documentation
+import docs from './tabs.docs.mdx';
 
 export default {
   title: 'Components/Tabs',
   parameters: {
+    docs: {
+      page: docs
+    },
     layout: "padded",
     previewTabs: {
       canvas: {hidden: false}
     }
   }
 };
 
-export const all = () => {
-  return tabsHTML;
-};
\ No newline at end of file
+export const tabsOutline = () => {
+  useEffect(() => {
+    new Tabs({container: document.getElementById('tabsDefaultExample')});
+  });
+  return `
+  <div id="tabsDefaultExample" class="tabs tabs--outline">
+    <div class="tabs__tab-list" role="tablist" aria-label="Tab List">
+      <button class="tabs__tab tabs__tab--active" role="tab" aria-selected="true" aria-controls="tab1-panel" id="tab1">Tab 1</button>
+      <button class="tabs__tab" role="tab" aria-selected="false" aria-controls="tab2-panel" id="tab2" tabindex="-1">Tab 2</button>
+      <button class="tabs__tab" role="tab" aria-selected="false" aria-controls="tab3-panel" id="tab3" tabindex="-1">Tab 3</button>
+      <a class="tabs__link" href="#">Tab link</a>
+    </div>
+    <div class="tabs__tab-panel tabs__tab-panel--active" role="tabpanel" id="tab1-panel" aria-labelledby="tab1" tabindex="0">
+      <p>Tab1 panel text</p>
+    </div>
+    <div class="tabs__tab-panel" role="tabpanel" id="tab2-panel" aria-labelledby="tab2" tabindex="0" hidden="">
+      <p>Tab2 panel text</p>
+    </div>
+    <div class="tabs__tab-panel" role="tabpanel" id="tab3-panel" aria-labelledby="tab3" tabindex="0" hidden="">
+      <p>Tab3 panel text</p>
+    </div>
+  </div>
+  `;
+};
+
+tabsOutline.storyName = 'Default / Outline';
+
+export const tabsMinimal = () => {
+  useEffect(() => {
+    new Tabs({container: document.getElementById('tabsMinimalExample')});
+  });
+  return `
+  <div id="tabsMinimalExample" class="tabs">
+    <div class="tabs__tab-list" role="tablist" aria-label="Tab List">
+      <button class="tabs__tab tabs__tab--active" role="tab" aria-selected="true" aria-controls="tab21-panel" id="tab21">Tab 1</button>
+      <button class="tabs__tab" role="tab" aria-selected="false" aria-controls="tab22-panel" id="tab22" tabindex="-1">Tab 2</button>
+      <button class="tabs__tab" role="tab" aria-selected="false" aria-controls="tab23-panel" id="tab23" tabindex="-1">Tab 3</button>
+    </div>
+    <div class="tabs__tab-panel tabs__tab-panel--active" role="tabpanel" id="tab21-panel" aria-labelledby="tab21" tabindex="0">
+      <p>Tab1 panel text</p>
+    </div>
+    <div class="tabs__tab-panel" role="tabpanel" id="tab22-panel" aria-labelledby="tab22" tabindex="0" hidden="">
+      <p>Tab2 panel text</p>
+    </div>
+    <div class="tabs__tab-panel" role="tabpanel" id="tab23-panel" aria-labelledby="tab23" tabindex="0" hidden="">
+      <p>Tab3 panel text</p>
+    </div>
+  </div>
+  `;
+};
+
+tabsMinimal.storyName = 'Minimal';
+
+export const tabsLined = () => {
```

Comment:
Active tab text doesn't line up with the inactive ones: 
![Screen Shot 2021-02-15 at 12 20 58 pm](https://user-images.githubusercontent.com/45442566/107898840-886cd000-6f88-11eb-9cb7-de354d718789.png)